Formula/ohdear-cli.rb

```diff
@@ -12,10 +12,10 @@ class OhdearCli < Formula
   depends_on "php"
 
   def install
-    bin.install "ohdear-cli.phar" => "ohdear-cli"
+    bin.install "ohdear.phar" => "ohdear"
```

Comment:
maybe add a symlink for `ohdear-cli`?

Reply:
Is there a good way to handle that in Brew, or just run `ln -s`? 🤷🏻

Reply:
check out `bin.install_symlink` in the repo.

Reply:
Also does upstream have preference one over another?

Reply:
The preference is to use `ohdear`, as that's the new name going forward. 👍🏻

Reply:
Thanks for those comments, that is pretty helpful. Also can you squash the commits? I think we are good to go after it :)